ivi-sdk-java/src/main/java/games/mythical/ivi/sdk/client/IVIOrderClient.java

```diff
@@ -156,6 +154,26 @@ public IVIFinalizeOrderResponse finalizeBitpayOrder(String orderId,
         return finalizeOrder(orderId, paymentData, fraudSessionId);
     }
 
+    public IVIFinalizeOrderResponse finalizeCybersourceOrder(String orderId,
+                                                             String cardType,
+                                                             String expirationMonth,
+                                                             String expirationYear,
+                                                             String instrumentId,
+                                                             String paymentMethodTokenId,
+                                                             String fraudSessionId) throws IVIException {
+        var paymentData = PaymentRequestProto.newBuilder()
+                .setCybersource(CybersourcePaymentRequestProto.newBuilder()
+                        .setCardType(cardType)
```

Comment:
actual hindsight now when it was done in Braintree, @cristian-t how do you feel with having to use `StringUtils.defaultString` when interacting with the SDK or would you like to be able to pass nulls.

Reply:
I already use defaultString, so I don't mind using it. And I'm always happy to avoid nulls